qualtran/bloqs/chemistry/pbc/first_quantization/prepare_t_test.py

```diff
@@ -33,15 +33,19 @@ def test_prepare_kinetic_t_counts():
     n_eta = (eta - 1).bit_length()
     expected_cost = (14 * n_eta + 8 * b_r - 36) + 2 * (2 * num_bits_p + 9)
     uni = UniformSuperpostionIJFirstQuantization(eta, num_bits_rot_aa=b_r)
-    _, counts = uni.call_graph()
-    qual_cost = counts[Toffoli()]
+    counts = get_cost_value(uni, QECGatesCost())
+    cost_dict = counts.total_t_and_ccz_count()
+    qual_cost = cost_dict['n_ccz']
     uni = UniformSuperpostionIJFirstQuantization(eta, num_bits_rot_aa=b_r).adjoint()
-    _, counts = uni.call_graph()
-    qual_cost += counts[Toffoli()]
+    counts = get_cost_value(uni, QECGatesCost())
+    cost_dict = counts.total_t_and_ccz_count()
```

Comment:
you can use `counts.total_toffoli_only` now, which saves a line and will complain if there are also T's and rotations

Reply:
Done./

Reply:
Done